packages/zowe-explorer/README.md

```diff
@@ -35,6 +35,8 @@ More information:
 
 > Zowe Explorer is compatible only with Theia 1.18.0 or higher.
 
+> Zowe Explorer could experience possible unexpected behaviors with the latest Theia releases.
```

Comment:
Should we remove the spacing between the notes ?
Not sure if that caused problems in some other builds before.

Reply:
it will get fixed once the other is in master then merged into this PR :slightly_smiling_face: